Source/Core/Core/HW/WiiSaveCrypted.cpp

```diff
@@ -337,8 +337,7 @@ void CWiiSaveCrypted::ImportWiiSaveFiles()
     }
     else
     {
-      std::string filename =
-          Common::EscapeFileName(reinterpret_cast<const char*>(file_hdr_tmp.name));
+      std::string filename = Common::EscapePath(reinterpret_cast<const char*>(file_hdr_tmp.name));
```

Comment:
Might make more sense than keeping it at "file name" - which is usually (at least for me) associated with just the name, and no path components (neither relative nor absolute). A brief comment wouldn't hurt either I guess...since other people (even Jos who probably has one of the best understandings of this stuff) may have a hard time considering all the possibilities without notes on what is going on.